src/test/java/emissary/output/DropOffUtilTest.java

```diff
@@ -601,6 +602,41 @@ void testExtractUniqueFileExtensions() {
         assertTrue(fileExts.contains("mp3"), "FILEXT values should contain \"mp3\"");
     }
 
+    @Test
+    void testExtractFullFilepaths() {
+        DropOffUtil util = new DropOffUtil();
+
+        final IBaseDataObject ibdo = new BaseDataObject();
+        ibdo.setParameter(FILE_ABSOLUTEPATH, "D:\\Users\\jdoe\\Documents\\Taxes 2023.csv");
```

Comment:
Use a loop or `@ParameterizedTest` for the nearly identical data-driven evaluation.

Also need to make sure we're testing at least one case for which both parameters have populated values; these all test the case for which FILE_ABSOLUTEPATH has a single value.

Should also add a "dot file" case (e.g., " /home/joe/.bashrc") and one for which the extension is longer than the configured max length.